lib/components/Toastr/index.js

```diff
@@ -42,13 +42,15 @@ const toastrList = new UniqueArray();
 const withUniqueCheck = (type, toastFunc) => (
   message,
   buttonLabel,
-  onClick
+  onClick,
+  customConfig = {},
```

Comment:
Thanks, Ajil for reviewing, I skipped that change considering it would be a breaking change. Apart from that, it's a good thing to implement. Let's see what's Amal's opinion.

Reply:
@amaldinesh7 _a could you please share your thoughts on this?

Reply:
@ajil-paul @deepakjosp Please check this video. Lemme know if you need more clarity on this.

cc: @ajmaln

Reply:
@amaldinesh7 looks good, going ahead with it.
cc: @ajil-paul @karthiknmenon

Reply:
@amaldinesh7 _a finished the implementation as you suggested. Could you please review the changes?

Reply:
The changes look good. Can you please add a case where only message and custom config are provided?

Move the custom config examples to a separate line in stories.

@deepakjosp _a

Reply:
@amaldinesh7  _a moved the stories to a separate line. Could you please re-review?